src/coreclr/vm/codeman.h

```diff
@@ -1711,9 +1942,18 @@ class EEJitManager : public IJitManager
     friend class CheckDuplicatedStructLayouts;
     friend class CodeHeapIterator;
 
-    VPTR_VTABLE_CLASS(EEJitManager, IJitManager)
+    VPTR_VTABLE_CLASS(EEJitManager, EECodeGenManager)
 
 public:
+    virtual ICorJitCompiler* GetCompiler()
```

Comment:
Is there a technical reason why we're not using `override` in cases like this where we're implementing an abstract virtual from the parent inside of a final class? Or is it just a style thing?

Reply:
Yes, adding it to even a single method triggers the compiler to require it everywhere in the class and if I remember it correctly, even on the base class. And I think it has some other ripple effect too which I cannot recall now.